news/env-dash-S.rst

```diff
@@ -0,0 +1,5 @@
+**Fixed:**
+
+* An error in the 'xon.sh' executable that only popped up during testing has
+  been fixed.  Specifically: It now calls '/usr/bin/env' with the '-S' option.
```

Comment:
It now doesn't call `env` at all ;)

Reply:
You're absolutely right :p Committed a fix for that as well.

Reply:
Thank you.